WordPress/src/main/res/layout/promote_with_blaze_card.xml

```diff
@@ -47,20 +47,20 @@
                 android:layout_height="wrap_content"
                 android:text="@string/promote_blaze_card_title"
                 android:textAlignment="viewStart"
-                android:textAppearance="@style/blaze_title"
+                android:textAppearance="@style/DashboardCardSubtitle"
                 app:layout_constraintEnd_toStartOf="@+id/my_site_promote_with_blaze_card_more"
                 app:layout_constraintStart_toStartOf="parent"
                 app:layout_constraintTop_toTopOf="parent"
                 tools:text="@string/promote_blaze_card_title" />
 
             <com.google.android.material.textview.MaterialTextView
-                android:id="@+id/my_site_promote_with_blaze_card_sub_title"
+                android:id="@+id/card_detail"
```

Comment:
np: It seems like the ids were following a naming pattern, so I am wondering why the name change here and not in the other instances? It seems out of place using two different patterns.

Reply:
Hey @zwarm , 
Good catch. This change was a mistake. I have reverted the change with the commit [8613fa0](https://github.com/wordpress-mobile/WordPress-Android/pull/18377/commits/8613fa0669ccd1b5e27d566d370dbb55cff88ec8)